contracts/evoting/controller/action.go

```diff
@@ -38,15 +43,22 @@ import (
 	sjson "go.dedis.ch/dela/serde/json"
 	"go.dedis.ch/kyber/v3/suites"
 	"golang.org/x/xerrors"
+	"go.dedis.ch/kyber/v3/util/key"
+
 )
 
 const token = "token"
 const inclusionTimeout = 2 * time.Second
 const contentType = "application/json"
 const getElectionErr = "failed to get election: %v"
 
+
+
 var suite = suites.MustFind("Ed25519")
 
+var kp = key.NewKeyPair(suite)
```

Comment:
This works for the scenario test, but since you use it directly in the proxy we should think of a way to adapt this key pair for the test or real life cases, especially since we don't have the private key used to sign the requests sent to the proxy in real elections (or do we?)